### PR TITLE
Cull back unneeded dependencies

### PR DESCRIPTION
**Pull request by Raynos**
This is a general cleanup PR to reduce the volume of dependencies

We currently have 44 dependencies and I would love to get that
to around ~20-25.

This PR brings it down to 36 and blocks on the bufrw PR

r: @jcorbin @kriskowal

### CODE REVIEW

**Comment by kriskowal** [comment]
:ok:

File: node/package.json [hunk 1]
```diff
@@ -19,16 +19,15 @@
...
+    "error": "^6.2.0",
     "farmhash": "^0.2.0",
-    "is-error": "2.0.0",
-    "json-stringify-safe": "5.0.0",
-    "null-logtron": "2.2.0",
+    "json-stringify-safe": "^5.0.0",
     "readable-stream": "1.0.33",
     "ready-signal": "^1.1.1",
-    "run-parallel": "^1.1.0",
+    "rezult": "^1.0.0",
```
Comment (jcorbin) [inline node/package.json]: so wait, that's strictly speaking a new dependency... we already had this in bufrw?

**Comment by jcorbin** [comment]
:-1:  there's way too much going on in this PR, please break it up
- dropping the logger and forcing all the use sites to branch is a big change
- maybe there's some other okay changes in here, but it's hard to see because of the logger change pervading
- shifting bufrw to a git link seems hasty, let's fix whatever the issue is that led you to that, heck let's even talk about wth the issue even was?

**Comment by Raynos** [comment]
Closing in favor of 4 other PRs.

**Pull request closed without merging**